Manager/MultilingualResources/Manager.ru.xlf

```diff
@@ -2561,1770 +2561,1770 @@ Unable to start the game.</target>
           <source>Failed to close CSGO, please do it manually and try again.
 Error: {0}.</source>
-          <target state="new">Failed to close CSGO, please do it manually and try again.
-Error: {0}.</target>
+          <target state="needs-review">Ошибка при закрытии CSGO, сделайте это вручную и попробуйте снова.
+Ошибка: {0}.</target>
         </trans-unit>
         <trans-unit id="DialogErrorKillingHlae" translate="yes" xml:space="preserve">
           <source>Failed to close HLAE, please do it manually and try again.
 Error: {0}.</source>
-          <target state="new">Failed to close HLAE, please do it manually and try again.
-Error: {0}.</target>
+          <target state="needs-review">Ошибка при закрытии HLAE, сделайте это вручную и попробуйте снова.
+Ошибка: {0}.</target>
         </trans-unit>
         <trans-unit id="DialogErrorStartingCsgo" translate="yes" xml:space="preserve">
           <source>Failed to start CSGO.
 Error: {0}.</source>
-          <target state="new">Failed to start CSGO.
-Error: {0}.</target>
+          <target state="needs-review">Ошибка при запуске CSGO.
+Ошибка: {0}.</target>
         </trans-unit>
         <trans-unit id="DialogInvalidDemo" translate="yes" xml:space="preserve">
           <source>Invalid demo file.</source>
-          <target state="new">Invalid demo file.</target>
+          <target state="needs-review">Неверный файл демки.</target>
         </trans-unit>
         <trans-unit id="DialogExportAllDemosConfirmation" translate="yes" xml:space="preserve">
           <source>Do you want to export only the selected demos or all the demos within the selected folder?</source>
-          <target state="new">Do you want to export only the selected demos or all the demos within the selected folder?</target>
+          <target state="needs-review">Вы хотите экспортировать только выбранные демки или все демки внутри выбранной папки?</target>
         </trans-unit>
         <trans-unit id="LabelAudioCodec" translate="yes" xml:space="preserve">
           <source>Audio codec</source>
-          <target state="new">Audio codec</target>
+          <target state="needs-review">Аудио кодек</target>
         </trans-unit>
         <trans-unit id="LabelVideoCodec" translate="yes" xml:space="preserve">
           <source>Video codec</source>
-          <target state="new">Video codec</target>
+          <target state="needs-review">Видео кодек</target>
         </trans-unit>
         <trans-unit id="LabelDate" translate="yes" xml:space="preserve">
           <source>Date</source>
-          <target state="new">Date</target>
+          <target state="needs-review">Дата</target>
         </trans-unit>
         <trans-unit id="ExportPlayersVoice" translate="yes" xml:space="preserve">
           <source>Export players voice</source>
-          <target state="new">Export players voice</target>
+          <target state="needs-review">Экспортировать голос игроков</target>
```

Comment:
Yep, need to build it.

Reply:
I found "Export players voice" in Context Menu. Tried it on Faceit demo, and yeah, It exports each player's voice 

So `Экспортировать голос игроков` sounds right

Reply:
In terms of use "Экспорт голосового чата" doing exact this thing: exporting ingame players voicemessages. So it's a voice-chat. In terms of meaning yeah, it indicates that players voices are exporting. If it is available to export only one player's voice instead of a whole voicechat, then yeah, "Экспортировать голос игрокА", maybe?

Reply:
> If it is available to export only one player's voice instead of a whole voicechat

Peeked at code and program, can't find anything like that

Вообще, мне стоит потренировать английский, но на всякий объясню так, т.к ещё туплю. Тут просто подходит буквальный перевод. Ведь при нажатии на кнопку создаётся аудиофайл под каждого игрока, который что-то говорил

Reply:
> Ведь при нажатии на кнопку создаётся аудиофайл под каждого игрока, который что-то говорил
Я больше смотрю в суть перевода. Если экспорт войсчата происходит в один файл, это это прямо "Экспорт голосового чата". Если все файлы разные, то "Экспорт голоса игроков"